lib/workers/repository/process/vulnerabilities.ts

```diff
@@ -171,69 +183,77 @@ export class Vulnerabilities {
           }
 
           logger.debug(
-            `Vulnerability ${vulnerability.id} affects ${packageName} ${depVersion}`
+            `Vulnerability ${osvVulnerability.id} affects ${packageName} ${depVersion}`
           );
-          const fixedVersion = this.getFixedVersion(
+          const fixedVersion = Vulnerabilities.getFixedVersion(
             ecosystem,
             depVersion,
             affected,
             versioningApi
           );
-          if (is.nullOrUndefined(fixedVersion)) {
-            logger.info(
-              `No fixed version available for vulnerability ${vulnerability.id} in ${packageName} ${depVersion}`
-            );
-            continue;
-          }
 
-          logger.debug(
-            `Setting allowed version ${fixedVersion} to fix vulnerability ${vulnerability.id} in ${packageName} ${depVersion}`
-          );
-          const rule = this.convertToPackageRule(
-            packageFileConfig,
-            dep,
+          vulnerabilities.push({
             packageName,
+            vulnerability: osvVulnerability,
```

Comment:
Idea is to differentiate between the OSV vulnerability and the new renovate vulnerability interface, which in the future do not have to be the same. Currently they are connected but in the future this will probably evolve in something more like we have with datasources.